docs/model_zoo.md

```diff
@@ -5,6 +5,7 @@ Scale hosts the following models in the LLM Engine Model Zoo:
 | Model Name            | Inference APIs Available | Fine-tuning APIs Available |
 | --------------------- | ------------------------ | -------------------------- |
 | `llama-7b`            | ✅                       | ✅                         |
+| `llama-7b-2`          | ✅                       | ✅                         |
```

Comment:
should this be `llama-2-7b`? on https://huggingface.co/meta-llama/Llama-2-7b it's that at least

Reply:
Ooooh good catch

Reply:
oh that's right